02_differential_expression/visu_DEG.R

```diff
@@ -0,0 +1,173 @@
+library(shiny)
+library(shinydashboard)
+library(DT)
+library(plotly)
+library(ComplexHeatmap)
+library(ggpubr)
+library(grid)
+library(purrr)
+library(dplyr)
+import::from(data.table, fread)
+
+
+deg_annotated <- fread("example_results/full_sample_type_normal_vs_tumor_pathways.csv") %>%
+  filter(padj < 0.05)
+deg_lists <- split(deg_annotated$genes, deg_annotated$pathway)
+deg_lists <- map(deg_lists, \(d) setdiff(strsplit(d, split = ",")[[1]], ""))
+geneset_list <- deg_lists[lengths(deg_lists) >= 5]
+
+volcano_data <- fread("example_results/full_sample_type_normal_vs_tumor_deg.csv") %>%
```

Comment:
add all needed files in the top with variables names so it can be easy changed by real-data